src/patternfly/components/Popover/popover.hbs

```diff
@@ -1,24 +1,23 @@
-<div class="pf-c-overlay">
-  <div class="pf-c-overlay__container {{direction}}">
-      <div class="pf-c-popover {{direction}}">
-        <div class="pf-c-popover__arrow"></div>
-        <div class="pf-c-popover__content pf-c-modal-box" role="dialog" aria-modal="true" aria-labelledby="modal-title" aria-describedby="modal-description">
-          <div class="pf-c-modal-box__close">
-            <button class="pf-c-button pf-m-action" aria-label="Close Dialog">
-              <i class="fas fa-times"></i>
-            </button>
-          </div>
-          <header class="pf-c-modal-box__header">
-            <h1 class="pf-c-modal-box__header-title" id="modal-title">
-              {{header}}
-            </h1>
-          </header>
-          <div class="pf-c-modal-box__body" id="modal-description">
-            {{> @partial-block}}
-          </div>
+<div class="Preview__overlay">
+  <div class="Preview__overlay-container {{direction}}">
```

Comment:
This might be a @dgutride question, but for similar cases where we need to add styles to the preview area to accommodate components with absolute-positioned elements, we have been putting classes in `examples/index.js`. 

We should only include classes for PF4 components in these files, since these files determine what code shows up in the code samples. But the classes you added here for the preview are for the workspace only, and are not something that we want consumers to copy when they copy the code samples.

Reply:
agree here - let's get it in as it is if we can and file an issue to address this on the workspace (for me to handle) @jgiardino

Reply:
Sounds good. I created this one: https://github.com/patternfly/patternfly-next/issues/526